loader/loader_test.go

```diff
@@ -238,6 +240,70 @@ var _ = Describe("Release Adapter", Ordered, func() {
 		})
 	})
 
+	When("calling GetPreviousRelease", func() {
+		var (
+			additionalRelease *v1alpha1.Release
+		)
+
+		BeforeEach(func() {
+			now := time.Now()
+
+			// Ensure the initial release is created
+			Eventually(func() error {
+				return k8sClient.Get(ctx, client.ObjectKey{Name: release.Name, Namespace: release.Namespace}, release)
+			}).Should(Succeed())
+
+			// Delay to ensure a distinct timestamp
+			time.Sleep(2 * time.Second)
+
+			additionalRelease = &v1alpha1.Release{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "additional-release",
+					Namespace:         "default",
+					CreationTimestamp: metav1.Time{Time: now.Add(-1 * time.Hour)},
```

Comment:
Wouldn't it be better to use release.metadata.creationtimestamp and subtract from that?